create_infotexts.py

```diff
@@ -53,13 +52,17 @@ def create_commons_filename(metadata, fotonr):
     :return: string
     """
     if not metadata[fotonr]["Beskrivning"] == "":
```

Comment:
You're right. My intention was actually to always enrich the description. 

Removed the if-else clause and left the TODO to handle the cases where there is no value in <Beskrivning>